kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkRollOverStrategy.java

```diff
@@ -1,6 +1,12 @@
 package com.slack.kaldb.chunkManager;
 
+import java.io.File;
+
 // TODO: Rename ChunkRollOverPolicyPredicate
```

Comment:
Remove this TODO.  Add a class doc for this interface.

Also, please add a new TODO to state that in future `ChunkRollOverStrategy` should take a chunk as an input and get statistics like message count, size etc.. from the chunk.